package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@material-ui/core": "^3.9.2",
+    "@material/typography": "^1.0.0",
```

Comment:
I don't believe this package is part of material-ui framework (v.3.9.2) (https://material-ui.com/style/typography/). 

We need to have a discussion on how to proceed with styling. I'm open to any cohesive solution, but at the moment we are using the following conflicting styling frameworks: 
- materialize-css
- material-ui v.20.2 (an out-of-date version of material-ui)
- material-design-icons (material-ui v3.9.2 uses a related packaged that turns Google's material design icons into React components. see @material-ui/icons: https://material-ui.com/getting-started/installation/)
- react-materialize (which seems to implement react components using materialize-css style sheets)
- styled-components (this library can be used as a way of overriding material-ui default styling, so probably okay to keep it)

Suggestion: remove @material/typography from code base since it's currently not used. @material-ui/core includes the necessary typography component (https://material-ui.com/api/typography/)